### ISSUE TITLE
GUI freezes on startup [Ubuntu/Linux] (Trac #128)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#132]
Starting with commit 1d0e30a, the GUI freezes on application startup on Linux. The Windows build is unaffected.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/128
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Starting with commit 1d0e30a, the GUI freezes on application startup on Linux. The Windows build is unaffected.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "GUI freezes on startup [Ubuntu/Linux]", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-10-22T16:40:25", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20121029T16:37:04: 
* ryan commented:

> Fixed in commit a01dcb815b7c59e334c3564d0bfe29b97b1aae6a.


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
